src/components/dev-hub/author-image.js

```diff
@@ -31,7 +31,7 @@ const AuthorImageContainer = styled('div')`
         ${({ gradientOffset, theme }) =>
             createShadowElement(
                 theme.gradientMap.greenTealOffset,
-                size.large,
+                '50%',
```

Comment:
any objections to changing this? if i understand correctly, this would allow image containers of all sizes to be circle-shaped (given that its width and height are equal). i think we should also be matching the border-radius of `CircularImage`? - https://github.com/mongodb/devhub/blob/staging/src/components/dev-hub/author-image.js#L46